deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -159,6 +159,42 @@ spec:
             online:
               description: Should the server be online?
               type: boolean
+            rootDeviceHints:
+              description: What user defined root device hints is used for this host?
+                User can optionally define the content of the root device hints
+              properties:
+                devicename:
+                  description: A device name like "/dev/vda"
+                  type: string
+                hctl:
+                  description: A SCSI bus address like 0:0:0:0
+                  type: string
+                model:
+                  description: Device identifier
+                  type: string
+                rotational:
+                  description: Device rotational type
+                  type: string
+                serial:
+                  description: Disk serial number
+                  type: string
+                size:
+                  description: Size of the device in GiB
+                  type: integer
+                vendor:
+                  description: Device vendor
+                  type: string
+                wwn:
+                  description: Unique storage identifier
+                  type: string
+                wwnvendorextention:
+                  description: Uunique vendor storage identifier
+                  type: string
+                wwnwithextention:
```

Comment:
wwn_with_extension

Reply:
It should always be camel case in k8s, wwwWithExtension as @zaneb mentioned above

Reply:
Fixed